src/DotLiquid/Util/StrFTime.cs

```diff
@@ -18,85 +20,197 @@ public static class StrFTime
+            { "v", (dateTime) => dateTime.ToString("%d-MMM-yyyy", CultureInfo.CurrentCulture).ToUpper().PadLeft(11, ' ') },
+            { "V", (dateTime) => dateTime.GetIso8601WeekOfYear("V") },
             { "w", (dateTime) => ((int) dateTime.DayOfWeek).ToString() },
-            { "x", (dateTime) => dateTime.ToString("d", CultureInfo.CurrentCulture) },
-            { "X", (dateTime) => dateTime.ToString("T", CultureInfo.CurrentCulture) },
+            { "W", (dateTime) => CultureInfo.CurrentCulture.Calendar.GetWeekOfYear(dateTime, CalendarWeekRule.FirstFullWeek, DayOfWeek.Monday).ToString().PadLeft(2, '0') },
+            { "x", (dateTime) => dateTime.ToString("MM/dd/yy", CultureInfo.CurrentCulture) },
+            { "X", (dateTime) => dateTime.ToString("HH:mm:ss", CultureInfo.CurrentCulture) },
             { "y", (dateTime) => dateTime.ToString("yy", CultureInfo.CurrentCulture) },
             { "Y", (dateTime) => dateTime.ToString("yyyy", CultureInfo.CurrentCulture) },
+            { "z", (dateTime) => dateTime.ToString("%K", CultureInfo.CurrentCulture).Replace(":", string.Empty) },
+            { ":z", (dateTime) => dateTime.ToString("%K", CultureInfo.CurrentCulture) },
+            // ::z - not implemented
             { "Z", (dateTime) => dateTime.ToString("zzz", CultureInfo.CurrentCulture) },
-            { "%", (dateTime) => "%" }
+            { "%", (dateTime) => "%" } // A % sign
         };
 
         private static readonly Dictionary<string, DateTimeOffsetDelegate> OffsetFormats = new Dictionary<string, DateTimeOffsetDelegate>
         {
             { "s", (dateTimeOffset) => ((long)(dateTimeOffset - new DateTimeOffset(1970, 1, 1, 0,0,0, TimeSpan.Zero)).TotalSeconds).ToString() },
+            { "z", (dateTimeOffset) => dateTimeOffset.ToString("%K", CultureInfo.CurrentCulture).Replace(":", string.Empty) },
+            { ":z", (dateTimeOffset) => dateTimeOffset.ToString("%K", CultureInfo.CurrentCulture) },
             { "Z", (dateTimeOffset) => dateTimeOffset.ToString("zzz", CultureInfo.CurrentCulture) },
         };
 
+        /// <summary>
+        /// Applies formatting consistent to the rules specified by the Ruby Time.strftime function.
+        /// The following exceptions apply;
+        /// - `%N` (Not standard) - DateTime does not support nanoseconds precision, so the default is microsecond (6dp).
+        /// - `%9N` (Not implemented) - DateTime does not support nanoseconds precision.
+        /// - `%Z` (Not standard) - DotLiquid returns an offset (+00:00), Ruby states this should return a timezone description (e.g. 'GMT')
+        /// <see href="https://help.shopify.com/themes/liquid/filters/additional-filters#date"/>
+        /// <see href="https://ruby-doc.org/core-3.0.0/Time.html#method-i-strftime"/>
+        /// </summary>
+        /// <param name="dateTime"/>

```

Comment:
Best to keep param documentation consistent. Ideally provide a text description for each parameter. If we omit the text description, then at least we should be consistent on how we omit it (i.e. self-closing XML element vs explicit open-close element, vs no element at all)

Reply:
Standardized all <param /> elements with descriptions.